README.rst

```diff
@@ -53,12 +57,52 @@ Main Package Classes
 6. **fastq.Reader & fastq.Record** fastq reader and fastq record class that exposes the fastq fields
    as a lightweight, lazy-parsed python object.
 
+7. **Metrics** calculate information about the genes and cells of an experiment
+
+8. **Bam** Split bam files into chunks and attach barcodes as tags
+
 
 Viewing Test Results and Coverage
 ---------------------------------
+
 to calculate and view test coverage cd to the ``sctools`` directory and
 type the following two commands to generate the report and open it in your web browser:
 
 .. code bash
    pytest --cov-report html:cov_html --cov=sctools
    open cov_html/index.html
+
```

Comment:
It looks like the above block is not working in the Github rendered readme page?